openlibrary/catalog/add_book/load_book.py

```diff
@@ -165,6 +165,12 @@ def get_redirected_authors(authors: list["Author"]):
         # Always match on OL ID, even if remote identifiers don't match.
         return get_redirected_authors([record])
 
+    # Validate that the author name is not empty
+    if not author.get("name", "").strip():
```

Comment:
Move this under the `remote_ids` section ; we'll want `remote_ids` to still match, even if a name is not provided.